<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>회원가입</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>경기천년제목 Medium</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>이름</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>경기천년제목 Medium</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>학번</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>경기천년제목 Medium</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>비밀번호</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="join_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>회원가입</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="name_text_edit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>256</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="pw_text_edit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>256</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="id_text_edit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>256</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
     </font>
    </property>
    <property name="text">
     <string>※ 원본 사진은 저장되지 않습니다.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="home_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>홈</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>join_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>complete_join()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>complete_join()</slot>
 </slots>
</ui>
